### ISSUE TITLE
What is this repo for?

### ISSUE THREAD

**Issue by yeya24** [dragonflyoss/dragonfly-examples#1]
Demo purpose or tutorial? 
I think this is a good example for configuring Dragonfly.
Can we provide more docs?
@lowzj

**Comment by lowzj** [comment]
Yes, we can! I think we can provide different examples for different version.
